rest_docs/rest-catalog-open-api-v0.1.yaml

```diff
@@ -0,0 +1,727 @@
+        # or if a client or server has been misconfigured or some bug exists
+        # and the client is actually calling endpoints that the server doesn't
+        # have.
+        #
+        # We can consider a different response code to use than 412.
+        '412':
+          description: Table to rename from does not exist
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NoSuchTableError'
+              example:
+                '{ error: { message: "Table does not exist", type: "NoSuchTableException", code: 41202 }'
+        '409':
+          description: The new table identifier, the to table rename to, already exists.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TableAlreadyExistsError'
+              example:
+                '{ error: { message: "Namespace already exists", type: "AlreadyExistsException", code: 40902 }'
+  /v1/namespaces/{namespace}/properties:
+    parameters:
+      - name: namespace
+        in: path
+        required: true
+        schema:
+          type: string
+    get:
+      tags:
+        - Catalog API
+      summary: Load the metadata properties for a namespace
+      operationId: loadNamespaceMetadata
+      description: Return all stored metadata properties for a given namespace
+      responses:
+        '200':
+          description: OK
+          content:
+            application:
+              schema:
+                $ref: '#/components/schemas/GetNamespaceResponse'
+        '417':
+          description: Namespace not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NoSuchNamespaceError'
+              example:
+                '{ error: { message: "Namespace does not exist", type: "NoSuchNamespaceException", code: 41701 }'
+    put:
+      tags:
+        - Catalog API
+      summary: Add or overwrite properties to an existing namespace
+      operationId: setNamespaceProperties
+      description:
+        Adds propertiess for a namespace. This will overwrite any existing properties,
+        and merge with the others.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RemovePropertiesRequest'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            'application/json':
+              schema:
+                type: boolean
+        '417':
+          description: Namespace not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NoSuchNamespaceError'
+              example:
+                '{ error: { message: "Namespace does not exist", type: "NoSuchNamespaceException", code: 41701 }'
+        '409':
+          description: Namespace already exists
```

Comment:
Looks like a copy & paste problem. Why have `NoSuchNamespaceException` and `NamespaceAlreadyExists` exception? Either it should or shouldn't exist before using this route.